### ISSUE TITLE
Sound and ringtone selection not working properly in Alarms (or in Profiles settings for sms, im, emails and calls)

### ISSUE THREAD

**Issue by Arno1103** [maemo-leste/bugtracker#721]
From Clock app (or Clock & Alarms applet), it is ONLY possible to select another ringtone from Clock title / Alarm settings, not from Alarm menu itself.
And sound works only if you select a ringtone/music from MyDocs directory.

The issue seems general and a bit different in Profiles settings: i can't select another sound or ringtone from the list in /usr/share/sounds. If i click on 'more' BTN, i'm able to select a sound from MyDocs directory.
But sound works randomly.

**Comment by IMbackK** [comment]
yes the problem is that the soundfiles and metadata are hardcoded in libprofile for no reason and the sound files hardcoded there, we dont have as we lack copywrite

**Comment by Arno1103** [comment]
For ringtones working randomly, the issue seems only due to wrong permissions: using chmod +r with ringtones in MyDocs/.sounds solved the issue on my N900. The ringtone and sphone caller UI start now very quickly when the other side makes the call.
I have to doublecheck with SMS's and IM.

**Comment by Arno1103** [comment]
Sounds work normally for SMS and IM now.
